lib/mayaUsd/ufe/UsdTransform3dMatrixOp.cpp

```diff
@@ -128,105 +131,64 @@ class UsdSetMatrix4dUndoableCmd : public UsdUndoableCommand<Ufe::SetMatrix4dUndo
     const Ufe::Matrix4d _newM;
 };
 
-// Helper class to factor out common code for translate, rotate, scale
-// undoable commands.
-class UsdTRSUndoableCmdBase
+// Factor out common code for translate, rotate, scale undoable commands.
+class MatrixOpUndoableCmdBase : public UsdSetXformOpUndoableCommandBase<GfMatrix4d>
```

Comment:
Integration of new base class for matrix op Transform3d.  Again, new command code is simpler than the value-based code it replaces.